CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Updates should follow the [Keep a CHANGELOG](https://keepachangelog.com/) princi
 
 ## [Unreleased][unreleased]
 
+## [0.3.15] - 2022-02-02
```

Comment:
Thanks for adding this! Please also add a reference to the new release on lines 251 & 252.